examples/mp3.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/bobertlo/go-mpg123/mpg123"
+	"github.com/gordonklaus/portaudio"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("missing required argument:  input file name")
+		return
+	}
+	fmt.Println("Playing.  Press Ctrl-C to stop.")
+
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt, os.Kill)
+
+	// create mpg123 decoder instance
+	decoder, err := mpg123.NewDecoder("")
+	chk(err)
```

Comment:
I didn't run this exact code no. That's not possible because `main()` is defined in all files in the examples dir 😏 

But I should indeed have double checked this example by copying the code to a different dir and test it. Will do that and update in a bit...